.changeset/strange-pillows-argue.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/math-input": patch
+---
+
+Fix dom errors from SVG markup and adjust stories to remove warnings
```

Comment:
**thought**: I don't think we need a changeset for this change.  My reasoning for whether I need to do a Changeset is if the change affects consumers of this package. I guess the DOM errors could be relevant to consumers, but not the stories change.

Reply:
Isn't that what patches are for? Perseus CI yells at me if there is no changeset

Reply:
how I understand it:
Path = code change with no apparent change for user
Minor = additional, compatible features that do not affect current consumers
Major = non-compatible changes to API that need attention from current consumers

Reply:
how do we interpret semver in this codebase?

Reply:
Your understanding of semver is correct and we use that interpretation for Perseus as well.  But the difference I'm pointing out is that changsets are a way to communicate with world outside of this package (they form the basis of the changelogs that we publish). For internal changes, I think the more standard way to see what changed would be to just look at the PRs included in a release. 

You can always use `yarn changeset --empty` to create a changeset file that is empty that will satisfy the Changeset Github Action.

Reply:
Oh, gotcha! Noted.